content/doc/book/system-administration/admin-password-reset.adoc

```diff
@@ -0,0 +1,32 @@
+---
+layout: section
+---
+ifdef::backend-html5[]
+:notitle:
+:description:
+:author:
+:email: jenkinsci-users@googlegroups.com
+:sectanchors:
+:toc: left
+endif::[]
+
+==Admin-Password-Reset Jenkins
+As a system administrator, you may need to reset the Jenkins admin password if you have lost or forgotten it. To cover this possibility we have steps to follow to reset the admin password.
+
+==Steps to reset Jenkins Admin Password
+* Login to your Jenkins server and edit the Jenkins configuration file(config.xml) inside your Jenkins folder.
+* Look for the **useSecurity** option and change it from **true** to **false**.
+* Save your file and close it.
+* Restart Jenkins service to apply changes.(This time it will not ask for password)
+* On the dashboard click on **Manage Jenkins** option on the left side.
+* In Manage Jenkins , in the **Security** section you will get **Configure Global Security**. Click on it.
```

Comment:
```suggestion
* In the *Security* section, select *Configure Global Security*.
```
suggestion for wording/instructions to be clear, remove unnecessary instruction at beginning of sentence